.github/workflows/gradle-run-ui-tests.yml

```diff
@@ -50,15 +50,13 @@ jobs:
           disable-animations: false
           script: echo "Generated AVD snapshot for caching."
 
-      - name: Test with Gradle
-        run: ./gradlew runUnitTests
-
       - name: Android Instrumentation Tests
         uses: reactivecircus/android-emulator-runner@v2
         with:
           api-level: {{ matrix.api-level }}
           script: ./gradlew connectedCheck
 
+        # TODO: verify route for android tests
       - name: Publish Unit Test Results
         uses: EnricoMi/publish-unit-test-result-action/composite@v1.25
         if: always()
```

Comment:
If above says UI tests, why are we running also the publishing of the unit tests in this action?

Reply:
In fact, it's a leftover from the original file, now we are going to run `unit test` and `ui test` on different workflows, achieving parallelism =)